pkg/platform/kube/monitoring/function.go

```diff
@@ -146,49 +150,68 @@ func (fm *FunctionMonitor) updateFunctionStatus(function *nuclioio.NuclioFunctio
 		stateChanged = true
 	}
 
-	if stateChanged {
-		fm.logger.InfoWith("Updating function",
+	// log and return if function did not change
+	if !stateChanged {
+		fm.logger.DebugWith("Function state did not change",
```

Comment:
Redundant log - monitoring should have very very minimal logging - only when things change or fail otherwise it will spam